User Interface/RPG Interface/source/State_Title.hx

```diff
@@ -66,6 +66,15 @@ class State_Title extends FlxUIState
 		}		
 	}	
 	
+	public override function update(elapsed:Float):Void
```

Comment:
Yup, that's debug code. And the reason I added this instead of just using "vc" was b/c for a while the console was broken when testing with -Dnext. Anyways, I'm removing this function as it's unecessary.